config/v2/config.go

```diff
@@ -183,12 +189,21 @@ type DatadogProvider struct {
 	Version *string `yaml:"version,omitempty"`
 }
 
+type SentryProvider struct {
+	Version *string `yaml:"version,omitempty"`
+	BaseUrl *string `yaml:"base_url,omitempty"`
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>struct field `BaseUrl` should be `BaseURL` (golint)